php/octo-facts/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.7"
+services:
+  app:
+    image: octofacts
+    restart: unless-stopped
+    working_dir: /app
+    user: laravel
+    volumes:
+      - .:/app
+    networks:
+      - wolfi
+
+  nginx:
+    image: cgr.dev/chainguard/nginx
+    restart: unless-stopped
+    ports:
+      - 8000:8080
+    volumes:
+      - .:/app
+      - ./nginx.conf:/etc/nginx/nginx.conf
+    networks:
+      - wolfi
+
+  mariadb:
+    image: cgr.dev/chainguard/mariadb
+    restart: unless-stopped
+    environment:
+      MARIADB_ALLOW_EMPTY_ROOT_PASSWORD: 1
+      MARIADB_USER: laravel
+      MARIADB_PASSWORD: password
```

Comment:
I know it's just an example and this isn't sensitive, but it might be worth keeping out of the example as people will copy it. You could use an external env var or external file e.g. https://docs.docker.com/compose/use-secrets/

Reply:
Good observation. I can add a note about it, because this is not to be used for production at all. The password is also hardcoded in the `.env` file, which is a normal practice for Laravel dev environments - the safeguard they have in place is keeping the `.env` in the `.gitignore` file so that it's not pushed to a public repo by accident. To keep this in place I added the .env.dev file to be copied into a `.env` file.

Reply:
I added this note to the guide:

>Please note this Docker Compose setup is intended for local development only. For production environments, you should never keep sensitive data like database credentials in plain text. Check the [Docker Compose Documentation](https://docs.docker.com/compose/use-secrets/) for more information on how to handle sensitive data in Compose files.

Reply:
Great!